File: iniciante/python/1009-salario-com-bonus.py
def salario(vendedor, salario_fixo, total_vendas):
    comissao_ganha = total_vendas * 0.15
    salario_total = comissao_ganha + salario_fixo
    return print(f'TOTAL = R$ {salario_total:.2f}')


nome_vendendor = input()
sal_fixo = float(input())
vendas_total = float(input())

salario(nome_vendendor, sal_fixo, vendas_total)

File: iniciante/c/1009-salario-com-bonus.c
#include <stdio.h>

int main(){
    char nome_vendedor;
    double salario_fixo, total_vendas, salario;
    scanf("%s %lf %lf", &nome_vendedor, &salario_fixo, &total_vendas);
    salario = salario_fixo + total_vendas * 0.15;
    printf("TOTAL = R$ %.2lf\n", salario);
    return 0;
}
